### PR TITLE
Remove references to Elmo

### PR DESCRIPTION
**Pull request by flodolo**
Waiting until Elmo is actually shut down.

### CODE REVIEW

File: src/products/firefox_desktop/adding_nightly.md [hunk 1]
```diff
@@ -4,18 +4,16 @@
...
-* Application: leave Firefox.
-* Version: leave the default.
-
-Click the *Create Buglinks* button, and a set of links will appear at the bottom of the page. Not all of them might be needed, for example LDAP is required only for Tableau access, repositories might already exist, etc. Make sure to always start with the `shipping` link, since that’s going to be the tracker bug, with an alias used in other bugs to set dependencies (`fx-l10n-LOCALECODE`, `fx-l10n-lo` in this example).
-
-Always double check the content of the bugs for errors or outdated content. If the information is obsolete, you can update the templates in this [Wiki page](https://wiki.mozilla.org/L10n:Bugogram) (it will require a new deployment of Elmo to production to update the code generating the links).
+A list of bug templates is available in this [Wiki page](https://wiki.mozilla.org/L10n:Bugogram). There is currently no automation to help filing them, although search and replace could be used on some variables across the templates.
+
+Let’s consider `Lao (lo)` as an example:
+* Locale (`%(loc)s`): `lo`.
```
Comment (Delphine) [inline src/products/firefox_desktop/adding_nightly.md:10]: why is there a "c" there? Maybe I'm confused...

File: src/products/firefox_desktop/adding_pontoon.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Adding a new locale to Pontoon
 
 * If the locale is not [available](https://pontoon.mozilla.org/teams/) in Pontoon yet, [consult this document](../../tools/pontoon/adding_new_locale.md) for instructions on how to add it.
-* File a bug to create a new Mercurial repository in [l10n-central](https://hg.mozilla.org/l10n-central/). Several bug templates, including one to request a new repository, are available in [this page](https://l10n.mozilla.org/bugs/new-locale). For more information about these templates, [refer to this document](adding_nightly.md).
+* File a bug to create a new Mercurial repository in [l10n-central](https://hg.mozilla.org/l10n-central/). Several bug templates, including one to request a new repository, are available in [Wiki page](https://wiki.mozilla.org/L10n:Bugogram). For more information about these templates, [refer to this document](adding_nightly.md).
```
Comment (Delphine) [inline src/products/firefox_desktop/adding_pontoon.md]: Nit: "in THIS wiki page"

File: src/products/firefox_desktop/adding_nightly.md (hunk 1 above)
Comment (flodolo) [reply src/products/firefox_desktop/adding_nightly.md:10]: It's the name of the variable (`loc` as in `locale`, not `lo` for Lao)

**Review by Delphine — APPROVED** [review]
LGTM now!

**Pull request merged**